lib/bolt/inventory.rb

```diff
@@ -45,7 +45,7 @@ def self.from_config(config)
         begin
           data = YAML.safe_load(ENV[ENVIRONMENT_VAR])
         rescue Psych::Exception
-          raise Bolt::CLIError, "Could not parse inventory from $#{ENVIRONMENT_VAR}"
+          raise Bolt::Error, "Could not parse inventory from $#{ENVIRONMENT_VAR}"
```

Comment:
Should this have an issue code?

Reply:
issue code is a puppet thing that we don't really make use of.
This should happen before we get into puppet anyway.

Reply:
Oh, I meant kind, not issue code. I think this might straight-up fail right now, because it's only passing 1 argument but `Bolt::Error` requires 2.

Reply:
ah yeah that will be an argument error